tests/models/test_models_vae.py

```diff
@@ -173,6 +173,56 @@ def test_output_pretrained(self):
         self.assertTrue(torch_all_close(output_slice, expected_output_slice, rtol=1e-2))
 
 
+class AsymmetricAutoencoderKLTests(ModelTesterMixin, UNetTesterMixin, unittest.TestCase):
+    model_class = AsymmetricAutoencoderKL
+    main_input_name = "sample"
+    base_precision = 1e-2
+
+    @property
+    def dummy_input(self):
+        batch_size = 4
+        num_channels = 3
+        sizes = (32, 32)
+
+        image = floats_tensor((batch_size, num_channels) + sizes).to(torch_device)
+        mask = torch.ones((batch_size, 1) + sizes).to(torch_device)
+
+        return {"sample": image, "mask": mask}
+
+    @property
+    def input_shape(self):
+        return (3, 32, 32)
+
+    @property
+    def output_shape(self):
+        return (3, 32, 32)
+
+    def prepare_init_args_and_inputs_for_common(self):
+        init_dict = {
+            "in_channels": 3,
+            "out_channels": 3,
+            "down_block_types": ["DownEncoderBlock2D", "DownEncoderBlock2D"],
+            "down_block_out_channels": [32, 64],
+            "layers_per_down_block": 1,
+            "up_block_types": ["UpDecoderBlock2D", "UpDecoderBlock2D"],
+            "up_block_out_channels": [32, 64],
+            "layers_per_up_block": 1,
+            "act_fn": "silu",
+            "latent_channels": 4,
+            "norm_num_groups": 32,
+            "sample_size": 32,
+            "scaling_factor": 0.18215,
+        }
+        inputs_dict = self.dummy_input
+        return init_dict, inputs_dict
+
+    def test_forward_signature(self):
+        pass
+
+    def test_forward_with_norm_groups(self):
+        pass
```

Comment:
Why are we skipping this?

Reply:
test_forward_signature is skipped also in AutoencoderKL, as it tests UNet signatures